src/lib/component/base/Banner.js

```diff
@@ -103,9 +101,9 @@ export const styles = theme => {
     },
 
     success: {
-      backgroundColor: green[600],
+      backgroundColor: theme.palette.success.secondary,
       "&::before": {
-        backgroundColor: green[600],
+        backgroundColor: theme.palette.success.secondary,
```

Comment:
I wonder if it would be simpler to instead fade the success and warning colours here. eg.

```javascript
import { fade } from "/vendor/@material-ui/core/styles/colorManipulator";
// ...
backgroundColor: fade(theme.palette.success, 0.2),
```

This way the success, warn, error, etc. colors could keep the styles simpler and usage more predictable.

Reply:
For warning specifically there are different colours. Normal vision uses an amber colour instead of yellow defined for the status icons. 

Success was also originally a different colour for banners and toasts because it was grey and hard to read. However one of my more recent commits changed it to a darker grey, so in that case it may work.